code/drasil-example/Drasil/GlassBR/Assumptions.hs

```diff
@@ -71,25 +71,25 @@ a4Desc mainIdea = foldlSent [S "The", plural value, S "provided in",
   makeRef (SRS.valsOfAuxCons SRS.missingP []), S "are assumed for the",
   phrase mainIdea, sParen (ch mainIdea) `sC` S "and the",
   plural materialProprty `sOf` foldlList (map ch
-  (take 3 assumptionConstants))]
+  (take 3 assumptionConstants))] +:+ S "[IM1, DD3, DD5, DD7, DD9]"
```

Comment:
Why are IM1, DD3, DD5, DD7, and DD9 hardcoded here?

Reply:
ditto for hard-coded values below.

Reply:
Make sure this is logged in issues or something as well (preferably pointing to the commit these references were added). If we include them without a pointer to them to say "this is wrong", we might forget about them.